{{cookiecutter.project_slug}}/.github/workflows/linting.yml

```diff
@@ -11,7 +11,7 @@ jobs:
           python-version: '3.10'
           cache: 'pipenv'
       - run: pipenv install --dev --deploy
-      - run: pipenv run black server
+      - run: pipenv run black --check server
```

Comment:
@OkayItsMikael Called this out before.
Black will run but it won't fail the PR. This fixes that.